src/test/java/org/stellar/sdk/MemoTest.java

```diff
@@ -112,7 +111,10 @@ public void testMemoHashInvalidHex() {
     @Test
     public void testMemoReturnHashSuccess() {
         MemoReturnHash memo = Memo.returnHash("4142434445464748494a4b4c");
-        assertEquals(MemoType.MEMO_RETURN, memo.toXdr().getDiscriminant());
+        org.stellar.sdk.xdr.Memo memoXdr = memo.toXdr();
```

Comment:
why not import `org.stellar.sdk.xdr.Memo` as a type directly (instead of referencing the full name here)?

Reply:
There's a class with the same name (`Memo`) in `org.stellar.sdk` package where this test file live. It would be a name conflict.

Reply:
👍